packages/devtools/devtools/package.json

```diff
@@ -41,7 +41,7 @@
     "phosphor-react": "^1.4.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.4.0",
```

Comment:
Did this upgrade come with something you needed for this PR, or did you just want to get us closer to 6.8.1 (latest)?

Reply:
I was having issues with the types of router and I saw that they added some new stuff in 6.4 so I thought that might help but it didn't. Turns out the type is only exported from `@remix-run/router` so I needed to add that as a dev dep and grab it from there. This bump is unnecessary but I decided just to leave it.